src/checks_windows.go

```diff
@@ -179,7 +179,10 @@ func commandOutput(commandGiven, desiredOutput string) (bool, error) {
 }
 
 func packageInstalled(packageName string) (bool, error) {
-	packageList := getPackages()
+	packageList, err := getPackages()
```

Comment:
```go
if packageList, err := getPackages(); err != nil {
    return false, err
}

Reply:
I think this would limit the scope of `packageList` to the `if`statement. We could do an `else` but I feel like that unnecessarily complicates the structure of the function.

Reply:
Wait isn't packageList not being used after?

Reply:
Oh wait

Reply:
Nvm